sycl/doc/extensions/README.md

```diff
@@ -38,7 +38,7 @@ DPC++ extensions status:
 | [Level-Zero backend specification](LevelZeroBackend/LevelZeroBackend.md)                                           	      | Supported                                 | |
 | [Platform Context](PlatformContext/PlatformContext.adoc) | Proposal | |
 | [SYCL_EXT_ONEAPI_DEVICE_IF](DeviceIf/device_if.asciidoc) | Proposal | |
-| [SYCL_INTEL_group_sort](GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc)                                                     | Proposal                                  | |
+| [SYCL_INTEL_group_sort](GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc)                                                     | Partially supported                                  | |
```

Comment:
Let's change this to "Partially supported (experimental)".

What is not yet supported?  Can we list the supported / unsupported features in the API document?  Otherwise, how will a user know what they can use?

Note that I do not want to add information about supported / unsupported features here in the README because this file is getting out-of-date w.r.t. to the specs themselves.  I think we will more likely keep the information up-to-date if it is all in the spec, rather than this separate README.

Reply:
> Let's change this to "Partially supported (experimental)".

Nit: Why not just "experimental"?  Going forward it might be better if we reserve "supported" for features that are implemented and supported in the sense that customers (of the product compiler) can expect paid support.

Reply:
That's OK too.  There is an existing example of another extension saying "supported (experimental)", which is why I suggested it.  Either way is fine with me as long as it says "experimental".

I expect to reconsider this README later when we reorg the specs.

Reply:
Seems we are agree to change the wording to "experimental"

Reply:
That change to the README looks good, but my question about "partially supported" remains.  Are there features in this spec that are not yet supported?

Reply:
Yes. Currently only sorting functions and default sorter is supported. Radix sorter is not implemented yet